src/Tribe/Main.php

```diff
@@ -596,4 +597,18 @@ public function embed_head() {
 		<link rel="stylesheet" id="tribe-tickets-embed-css" href="<?php echo esc_url( $css_path ); ?>" type="text/css" media="all">
 		<?php
 	}
+
+	/**
+	 * Enqueue and localize script for use in ticket delete alert dialogue
+	 *
+	 * @since 4.3
+	 */
+	public function add_ticket_deletion_alert() {
+		$url = Tribe__Tickets__Main::instance()->plugin_url . 'src/resources/js/ticket-delete-alert.js';
+		wp_enqueue_script( 'tribe_tickets_ticket_delete_alert', $url, array( 'jquery' ), Tribe__Tickets__Main::VERSION, true );
+		$deletion_data = array(
+			'alert' => __( 'Are you sure you want to delete this ticket?', 'event-tickets' ),
+		);
+		wp_localize_script( 'tribe_tickets_ticket_delete_alert', 'ticket_notices', $deletion_data );
```

Comment:
A better way of improving this script would be to instead of adding one more JS variable via `wp_localize_scripts` would be to use the a `data-attr` on the link to translate, it would give us a little bit more context, like the Ticket name.